stage-2/src/main/java/epam/test/automation/java_io/optional_task/OptionalTask.java

```diff
@@ -0,0 +1,102 @@
+package epam.test.automation.java_io.optional_task;
+
+import java.io.*;
+import java.util.*;
+
+public class OptionalTask {
+    public static void main(String[] args) {
+        //1. Создать и заполнить файл случайными целыми числами. Отсортировать содержимое файла по возрастанию.
+        Random random = new Random();
+        ArrayList<String> list = new ArrayList<>();
+        File numbers = new File("data" + File.separator + "optional_1.txt");
+        try (FileWriter fw = new FileWriter(numbers, false);) {
+            for (int i = 0; i < 20; i++) {
+                String line = String.valueOf(random.nextInt(100));
+                fw.write(line);
+                fw.write('\n');
+            }
+        } catch (IOException e) {
+            e.getMessage();
+        }
+        try (FileReader fr = new FileReader(numbers)) {
+            Scanner scan = new Scanner(fr);
+            while (scan.hasNextLine()) {
+                list.add(scan.nextLine());
+            }
+            list.sort(Comparator.comparing(Integer::valueOf));
+        } catch (IOException e) {
+            e.getMessage();
+        }
+        try (FileWriter fwSort = new FileWriter(numbers)) {
+            for (int i = 0; i < list.size(); i++) {
+                fwSort.write(list.get(i));
+                fwSort.write('\n');
+            }
+        } catch (IOException e) {
+            e.getMessage();
+        }
+
+        // 2.Прочитать текст Java-программы и все слова public в объявлении атрибутов и методов класса заменить на слово private.
+        File fileJava = new File("D:\\Test_automation\\stage-2\\src\\main\\java\\epam\\test\\automation\\java_io\\optional_task", "CheckClass.java");
```

Comment:
Please avoid hardcoded paths. It doesn't work on machines with other directory structure